### PR TITLE
Copyright year update (2019->2020)

### LINKED ISSUE #16405: Copyright and Class header reports 2019, and now it's 2020
See https://github.com/hazelcast/hazelcast/blob/master/checkstyle/ClassHeader.txt#L2

Building snapshot today gives
```
2020-01-03 15:48:53.849  INFO 1 --- [           main] c.h.system                               : [172.17.0.2]:5701 [dev] [4.0-SNAPSHOT] Hazelcast 4.0-SNAPSHOT (20200103 - a6ae3d2) starting at [172.17.0.2]:5701
2020-01-03 15:48:53.850  INFO 1 --- [           main] c.h.system                               : [172.17.0.2]:5701 [dev] [4.0-SNAPSHOT] Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
```

### PR DESCRIPTION
**Pull request by kwart**
Fixes #16405.

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/instance/impl/DefaultNodeExtension.java [hunk 1]
```diff
@@ -194,7 +194,7 @@ protected String constructBuildString(BuildInfo buildInfo) {
     private void printNodeInfoInternal(BuildInfo buildInfo, String build) {
         systemLogger.info(getEditionString() + " " + buildInfo.getVersion()
                 + " (" + build + ") starting at " + node.getThisAddress());
-        systemLogger.info("Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.");
+        systemLogger.info("Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.");
```
Comment (neilstevenson) [inline hazelcast/src/main/java/com/hazelcast/instance/impl/DefaultNodeExtension.java]: Shouldn't this be derived ?

Comment (kwart) [reply hazelcast/src/main/java/com/hazelcast/instance/impl/DefaultNodeExtension.java]: How do you mean derived? If the idea is to print the current year, I think we can't go this way due to copyright expiration reasons.

Comment (neilstevenson) [reply hazelcast/src/main/java/com/hazelcast/instance/impl/DefaultNodeExtension.java]: Ok. So really my comment was "could this be derived" and the answer is "no".

**Comment by mmedenjak** [comment]
Seems someone committed DeserializingCompletableFutureTest_CompletionStageTest with the old header :(

**Comment by kwart** [comment]
Unrelated test failures #16430 and #16431. Merging.

Thanks for the reviews.

**Pull request merged**